### DISCUSSION TITLE
Add more context menu options on hubs

### DISCUSSION THREAD

**Discussion by JonzaUK** [pannal/plex-for-kodi#157] [Ideas]
Currently the only option when bringing up the context menu on hub items is "Hide Hub"

Can we have the option the below options like modern Plex clients

- Mark as watched
- Mark as unwatched
- Play from the beginning 
- Resume (if part watched)
- Go to show
- remove from continue watching (this just hides from the continue watching hub until you watch it again)
- Add to watch list (once implemented) 

Example of Apple TV context menu
https://github.com/user-attachments/assets/8414d823-791e-4854-8bec-174ba3b315c2

If anyone else as any thoughts or extras for the context menu comment below.

**Comment by pannal** [comment]
0.8.0-beta3.1 contains mark played/unplayed for any video-related item on home: https://forums.plex.tv/t/pm4k-plexmod-for-kodi-18-19-20-21/481208/3046

> Play from the beginning
> Resume (if part watched)

You can press Play (on your remote) on an item on Home and if it has a resume point you can decide whether to resume or start from zero

> Go to show

Any item on home goes to the Show (or the Movie) if entered

For the other things: I'll check whether it's possible or makes sense.

**Reply by JonzaUK** [reply]
oh awesome :) to be fair you are right about the play button this works for me, might be more of an issue with people with simpler remotes... but don't know if it would be quicker then just clicking into the item and hitting play?

For me - remove from continue watching would be really nice... as allows me to remove stuff I've started without having to mark as unwatched or deleting the media...

**Comment by Kuchiru** [comment]
@pannal I accidentally hid the continue watching hub from said context menu, i havent been able to find an option to unhide it.
Is there a way to get it back?

**Reply by pannal** [reply]
Yes, long press on the "Home" "library".

**Reply by Kuchiru** [reply]
Thank you!
